meeteval/wer/wer/time_constrained.py

```diff
@@ -370,12 +381,53 @@ def sort_segments(s: TimeMarkedTranscript):
     )
 
 
+def sort_and_validate( segments, sort, pseudo_word_level_timing, name):
+    # Check that all timings are valid
+    if len(segments.transcript) != len(segments.timings):
+        raise ValueError(
+            f'Number of words does not match number of timings in {name}: '
+            f'{len(segments.transcript)} != {len(segments.timings)}'
+        )
+
+    for t in segments.timings:
+        if t[1] < t[0]:
+            raise ValueError(f'The end time of an interval must be larger than the start time. Found {t} in {name}')
+
+    prune = True
+
+    assert sort in (True, False, 'segment', 'word')
```

Comment:
`assert sort in (True, False, 'segment', 'word'), sort`

Reply:
Done